src/components/Divider.tsx

```diff
@@ -36,12 +40,19 @@ const Divider = ({ path, cssRaw }: { path: Path; cssRaw?: SystemStyleObject }) =
 
   useEffect(setStyle)
 
+  const isTableView = useSelector(state => {
+    const parentId = head(parentOf(path)) || head(rootedParentOf(state, path))
```

Comment:
`parentOf` and `rootedParent` do the same thing, so we don't need both.

The only difference is that `parentOf` only works if `path.length > 1`. `rootedParentOf` works on any path and can differentiate between different root contexts. I will update the JSDOC to make this more clear!